a11y-meta-display-guide/2.0/draft/guidelines/index.html

```diff
@@ -838,32 +836,33 @@ <h4>Examples</h4>
 						adoption.</p>
 
 					<aside class="example" title="Descriptive explanations">
+<p><span data-localization-id="navigation-intro" data-localization-mode="descriptive">Navigation features</span></p>
 						<ul>
 							<li data-localization-id="navigation-toc" data-localization-mode="descriptive">Contains a
 								table of contents that provides direct access to all chapters of the text via
 								links</li>
+<li data-localization-id="navigation-index" data-localization-mode="descriptive">An index
+								provides links to referenced entries</li>
 							<li data-localization-id="navigation-structural" data-localization-mode="descriptive"
 								>Contains book elements such as headings, tables, etc for structured navigation</li>
-							<li data-localization-id="navigation-index" data-localization-mode="descriptive">Index
-								provides links to item references</li>
-
 							<li data-localization-id="navigation-page-navigation" data-localization-mode="descriptive">A
-								page list enables users to navigate directly to pages from the identified print source
+								page list enables users to directly go to    pages from the identified print source
```

Comment:
Although split infinitives aren't the end of the world anymore, this one stuck out when I read it. I would drop "directly" from the sentence as it doesn't add anything to the meaning.